frontend/src/hocs/withExpandable/withExpandable.tsx

```diff
@@ -47,82 +48,85 @@ function withExpandable(
+                borderTopStyle: "none",
+                borderBottomStyle: "solid",
+                borderBottomColor: $expanded
+                  ? colors.lightGray
+                  : colors.transparent,
+                marginLeft: "0",
+                marginRight: "0",
+                marginTop: "0",
+                marginBottom: "0",
+                paddingLeft: "0",
+                paddingRight: "0",
+                paddingTop: $expanded ? "1em" : 0,
+                paddingBottom: 0,
+              }),
+              props: { className: "streamlit-expanderContent" },
             },
-          },
-          Header: {
-            style: ({ $disabled }) => ({
-              marginBottom: "0",
-              marginLeft: "0",
-              marginRight: "0",
-              marginTop: "0",
-              paddingLeft: "0",
-              backgroundColor: colors.transparent,
-              borderBottomColor: colors.grayLighter,
-              color: $disabled ? colors.disabledColor : colors.black,
-              borderTopStyle: "none",
-              paddingBottom: "0.5em",
-              paddingRight: "0",
-              paddingTop: "0.5em",
-              fontWeight: 500,
-              ":hover": {
-                borderBottomColor: colors.primary,
+            PanelContainer: {
+              style: {
+                marginLeft: "0 !important",
+                marginRight: "0 !important",
+                marginTop: "0 !important",
+                marginBottom: "0 !important",
+                paddingLeft: "0 !important",
+                paddingRight: "0 !important",
+                paddingTop: "0 !important",
+                paddingBottom: "0 !important",
               },
-            }),
-            props: { className: "streamlit-expanderHeader" },
-          },
-          ToggleIcon: {
-            style: ({ $disabled }) => ({
-              marginRight: ".5rem",
-              color: $disabled ? colors.disabledColor : colors.black,
-            }),
-          },
-          Root: {
-            props: {
-              className: classNames("streamlit-expander", { empty }),
             },
-          },
-        }}
-      >
-        <Panel title={label} key="panel">
-          <WrappedComponent {...componentProps} disabled={widgetsDisabled} />
-        </Panel>
-      </Accordion>
+            Header: {
+              style: ({ $disabled }) => ({
+                marginBottom: "0",
+                marginLeft: "0",
+                marginRight: "0",
+                marginTop: "0",
+                paddingLeft: "0",
+                backgroundColor: colors.transparent,
+                borderBottomColor: colors.lightGray,
+                color: $disabled ? colors.disabled : colors.black,
+                borderTopStyle: "none",
+                paddingBottom: "0.5em",
+                paddingRight: "0",
+                paddingTop: "0.5em",
+                fontWeight: 500,
```

Comment:
Two trains of thoughts:

1. Add everything to the theme so that someone can sort out the theme and clarify.
2. Add everything that feels themed to the theme and then leave components as they are specified.

Honestly, I am choosing 2 more and more because I want to keep the same look as depicted and it starts creating additional overhead to move things into the theme. Afterward, one can analyze the theme and all the `styled-components.ts` files to see if things are worthwhile to add to the theme.

Also to be fair, the theme doesn't _need_ to have everything either, especially for these one-off use cases.

Reply:
Isn't this all in our theme already? I was thinking of spacing and fontWeights and these values align with values in our theme. 

Though expandable might also get redesigned so I'm fine whichever way you decide.

Reply:
Sorry. I was thinking something else. I can do that. I won't change the "em" styles cause they are slightly different, but hopefully shouldn't be too hard.